lib/spark.js

```diff
@@ -15,16 +15,16 @@ export default class Spark {
 
   async getRetrieval () {
     console.log('Getting retrieval...')
-    const res = await this.#fetch('https://spark.fly.dev/retrievals', {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({
-        sparkVersion: SPARK_VERSION,
-        zinniaVersion: Zinnia.versions.zinnia
-      })
+    const res = await this.#fetch('https://spark.fly.dev/rounds/current', {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' }
     })
```

Comment:
After this change, SPARK nodes will no longer submit their version when asking for a job. That removes an opportunity allowing us to quickly disable outdated clients via spark-api (that we fully control). Instead, outdated clients will learn about being outdated only after they finish the retrieval and submit the measurement.

Is it worth looking for options on how to preserve the ability to reject outdated SPARK nodes early? I think it's not.

Reply:
I agree, that's a low return of investment